packages/design-system-react-native/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -116,8 +114,15 @@ describe('Checkbox', () => {
   });
 
   it('sets accessibility props and merges style', () => {
+    const fn = jest.fn();
     const { getByTestId } = render(
-      <Checkbox label="Accept" isSelected style={{ margin: 4 }} testID="chk" />,
+      <Checkbox
+        label="Accept"
+        isSelected
+        onChange={fn}
+        style={{ margin: 4 }}
```

Comment:
Previously we had a type error here but using `style?: StyleProp<ViewStyle>;` fixes this see types file